test/regressions/site/src/tests/Avatar/ImageAvatar.js

```diff
@@ -2,12 +2,13 @@
 
 import React from 'react';
 import Avatar from 'material-ui/Avatar';
+import image from './ImageAvatar.jpg';
 
 export default function ImageAvatar() {
   return (
     <Avatar
       alt="Remy Sharp"
-      src="https://s3.amazonaws.com/uifaces/faces/twitter/rem/73.jpg"
```

Comment:
Should solve the other flaky test. That was the simplest solution I could think of.